app/assets/stylesheets/general/_titles.scss

```diff
@@ -49,7 +49,21 @@ h6 {
 
 h1, h2, h3, h4, h5, h6 {
   .header_anchor {
-    text-decoration: none;
+    float: left;
+    margin-left: -20px;
+    width: 20px;
+    height: 1em;
+    line-height: 1;
+
+    background: transparent image-url("icons/icon-link.svg") no-repeat;
+    background-position: left center;
+
+    opacity: 0;
```

Comment:
I think we might need to use `display: none` to hide the button, otherwise you might accidentally hit it with touch controls